package.json

```diff
@@ -9,7 +9,6 @@
   "author": "MIT",
```

Comment:
The Author is the MIT?

Reply:
It was an error when creating the project, it has been corrected.